2021-12-15 Shiny shiny little star/app.R

```diff
@@ -0,0 +1,122 @@
+library(DT)
+library(highcharter)
+library(readxl)
+library(shiny)
+library(tidyr)
+library(dplyr)
+library(stringr)
+library(tidytext)
+library(googlesheets4)
+library(textdata)
+library(tidyr)
+library(lubridate)
+library(RColorBrewer)
+library(wordcloud2)
+library(tm)
+library(tidyverse)
+library(qdapRegex)
+library(reshape2)
+library(chron)

```

Comment:
We need to add any new ones to `install.R` so that the Binder environment installs them